src/main/scala/nl/knaw/dans/api/sword2/servlets/ServiceInitializer.scala

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (C) 2015-2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.api.sword2.servlets
+
+import java.io.File
+import java.net.URI
+import java.util.regex.Pattern
+import javax.servlet.{ServletContextEvent, ServletContextListener, ServletException}
+
+import ch.qos.logback.classic.LoggerContext
+import ch.qos.logback.classic.joran.JoranConfigurator
+import nl.knaw.dans.api.sword2.{DepositHandler, LdapAuthSettings, Settings, SingleUserAuthSettings}
+import org.apache.commons.configuration.PropertiesConfiguration
+import org.slf4j.LoggerFactory
+
+class ServiceInitializer extends ServletContextListener {
```

Comment:
This class certainly has some potential to be done in a more Scala-esc style using `Try` and for-comprehensions! Not a requirement, because it is quite self contained, but would be an interesting challenge (or potential candidate for the 'group-refactoring-session' that you were thinking of?)